### ISSUE TITLE
[Apple MacBook Air (M1, 2020)] No audio output after upgrade

### ISSUE THREAD

**Issue by pixrl** [AsahiLinux/asahi-audio#38]
OS: Fedora Linux Asahi Remix 39 (Workstation Edition) aarch64
Host: Apple MacBook Air (M1, 2020)
Kernel: 6.8.7-402.asahi.fc39.aarch64+16k

```
asahi-platform-metapackage.aarch64                     0-10.fc39                                       @copr:copr.fedorainfracloud.org:group_asahi:fedora-remix-scripts 
asahi-platform-metapackage-audio.aarch64               0-10.fc39                                       @copr:copr.fedorainfracloud.org:group_asahi:fedora-remix-scripts 
asahi-platform-metapackage-core.aarch64                0-10.fc39                                       @copr:copr.fedorainfracloud.org:group_asahi:fedora-remix-scripts 
asahi-platform-metapackage-firefox.aarch64             0-10.fc39                                       @copr:copr.fedorainfracloud.org:group_asahi:fedora-remix-scripts 
asahi-repos.noarch                                     1-11                                            @copr:copr.fedorainfracloud.org:group_asahi:fedora-remix-branding
fedora-asahi-remix-release.noarch                      39-5                                            @copr:copr.fedorainfracloud.org:group_asahi:fedora-remix-branding
fedora-asahi-remix-release-common.noarch               39-5                                            @copr:copr.fedorainfracloud.org:group_asahi:fedora-remix-branding
fedora-asahi-remix-release-identity-workstation.noarch 39-5                                            @copr:copr.fedorainfracloud.org:group_asahi:fedora-remix-branding
fedora-asahi-remix-release-workstation.noarch          39-5                                            @copr:copr.fedorainfracloud.org:group_asahi:fedora-remix-branding
fedora-asahi-remix-scripts.noarch                      20240424-1.fc39                                 @copr:copr.fedorainfracloud.org:group_asahi:fedora-remix-scripts 
kernel-16k.aarch64                                     6.6.3-411.asahi.fc39                            @copr_copr.fedorainfracloud.org_group_asahi_kernel               
kernel-16k.aarch64                                     6.6.3-414.asahi.fc39                            @copr:copr.fedorainfracloud.org:group_asahi:kernel               
kernel-16k.aarch64                                     6.8.7-402.asahi.fc39                            @copr:copr.fedorainfracloud.org:group_asahi:kernel               
kernel-16k-core.aarch64                                6.6.3-411.asahi.fc39                            @copr_copr.fedorainfracloud.org_group_asahi_kernel               
kernel-16k-core.aarch64                                6.6.3-414.asahi.fc39                            @copr:copr.fedorainfracloud.org:group_asahi:kernel               
kernel-16k-core.aarch64                                6.8.7-402.asahi.fc39                            @copr:copr.fedorainfracloud.org:group_asahi:kernel               
kernel-16k-modules.aarch64                             6.6.3-411.asahi.fc39                            @copr_copr.fedorainfracloud.org_group_asahi_kernel               
kernel-16k-modules.aarch64                             6.6.3-414.asahi.fc39                            @copr:copr.fedorainfracloud.org:group_asahi:kernel               
kernel-16k-modules.aarch64                             6.8.7-402.asahi.fc39                            @copr:copr.fedorainfracloud.org:group_asahi:kernel               
kernel-16k-modules-core.aarch64                        6.6.3-411.asahi.fc39                            @copr_copr.fedorainfracloud.org_group_asahi_kernel               
kernel-16k-modules-core.aarch64                        6.6.3-414.asahi.fc39                            @copr:copr.fedorainfracloud.org:group_asahi:kernel               
kernel-16k-modules-core.aarch64                        6.8.7-402.asahi.fc39                            @copr:copr.fedorainfracloud.org:group_asahi:kernel               
kernel-16k-modules-extra.aarch64                       6.6.3-411.asahi.fc39                            @copr_copr.fedorainfracloud.org_group_asahi_kernel               
kernel-16k-modules-extra.aarch64                       6.6.3-414.asahi.fc39                            @copr:copr.fedorainfracloud.org:group_asahi:kernel               
kernel-16k-modules-extra.aarch64                       6.8.7-402.asahi.fc39                            @copr:copr.fedorainfracloud.org:group_asahi:kernel               
kernel-headers.aarch64                                 6.8.7-402.asahi.fc39                            @copr:copr.fedorainfracloud.org:group_asahi:kernel               
mesa-dri-drivers.aarch64                               24.1.0~asahipre20240228-2.fc39                  @copr:copr.fedorainfracloud.org:group_asahi:mesa                 
mesa-filesystem.aarch64                                24.1.0~asahipre20240228-2.fc39                  @copr:copr.fedorainfracloud.org:group_asahi:mesa                 
mesa-libEGL.aarch64                                    24.1.0~asahipre20240228-2.fc39                  @copr:copr.fedorainfracloud.org:group_asahi:mesa                 
mesa-libGL.aarch64                                     24.1.0~asahipre20240228-2.fc39                  @copr:copr.fedorainfracloud.org:group_asahi:mesa                 
mesa-libgbm.aarch64                                    24.1.0~asahipre20240228-2.fc39                  @copr:copr.fedorainfracloud.org:group_asahi:mesa                 
mesa-libglapi.aarch64                                  24.1.0~asahipre20240228-2.fc39                  @copr:copr.fedorainfracloud.org:group_asahi:mesa                 
mesa-va-drivers.aarch64                                24.1.0~asahipre20240228-2.fc39                  @copr:copr.fedorainfracloud.org:group_asahi:mesa                 
mesa-vulkan-drivers.aarch64                            24.1.0~asahipre20240228-2.fc39                  @copr:copr.fedorainfracloud.org:group_asahi:mesa                 
uboot-images-armv8.noarch                              1:2024.04-4.fc39                                @copr:copr.fedorainfracloud.org:group_asahi:u-boot      
```

**Comment by pixrl** [comment]
Seems like after reboot the issue has been fixed for now

**Comment by chadmed** [comment]
Updating your system and rebooting should fix this. This was likely due to a bug with the HDMI audio implementation which has been fixed in `6.8.7-403.asahi.fc39.aarch64+16k`
